src/components/Debugger.tsx

```diff
@@ -169,6 +169,13 @@ const JwtPayloadSection = ({ claim, disclosureFrame, tabHandler, mode }: any) =>
         </PayloadHeader>
         <div style={{ display: 'flex' }}>
           <JwtPayload payload={claim} setPayload={tabHandler} mode={mode} type="claim" />
+          <div
+            style={{
+              width: '1px',
+              height: '100%',
+              backgroundColor: 'rgba(0, 0, 0, 0.2)',
+            }}
+          />
```

Comment:
```suggestion
          <div
            role="presentation"
            aria-label="border"
            style={{
              width: '1px',
              height: '100%',
              backgroundColor: 'rgba(0, 0, 0, 0.2)',
            }}
          />
```

Reply:
If we don't use CSS for borders,
adding a description to explain it's for borders seems like an easier option.